kolibri/plugins/learn/assets/test/views/learning-activity-bar.spec.js

```diff
@@ -5,7 +5,9 @@ import LearningActivityBar from '../../src/views/LearningActivityBar';
 
 jest.mock('kolibri.coreVue.componentSets.sync');
 function makeWrapper({ propsData } = {}) {
-  return mount(LearningActivityBar, { propsData });
+  // stubbing out KCircularLoader, as using the actual component led to errors related to
+  // Vue Composition API - stub may not be needed once we upgrade to Vue 2.7
+  return mount(LearningActivityBar, { propsData, stubs: ['KCircularLoader'] });
```

Comment:
Thansk @thanksameeelian for cleaning this up, I think that it got broken as result of my recent circular loader updates.